spec/controllers/annotations_controller_spec.rb

```diff
@@ -343,36 +347,36 @@
       end
 
       it 'returns a 200 response' do
-        put :update, { annotation: @new_annotation, layer_id: 'layer/1'}, :format => "json"
+        put :update, { annotation: @new_annotation, layer_id: ['layer/1']}, :format => "json"
```

Comment:
Should we break these out into its own function like post_to_create. Since post_to_create is shared we should either pick a style of doing the command in the it block or having a shared function called from the it block.

Reply:
I'll make it something akin to `update_anno(anno, layer)`.